### DISCUSSION TITLE
Cannot get slint to use the Qt backend

### DISCUSSION THREAD

**Discussion by eliothhhh** [slint-ui/slint#4980] [Q&A]
Hi! So I'm trying to use the Qt backend with slint and rust on Windows 11, and I can't seem to get it to work. I first added the `backend-qt` feature into my `cargo.toml`, then set `SLINT_BACKEND=Qt` environment variable with `std::env::set_var()`. I have Qt installed and have manually added `C:/Qt/6.6.2/bin/` and `C:/Qt/6.6.2/bin/qmake.exe` to the path. I don't really know what to do anymore :/

![image](https://github.com/slint-ui/slint/assets/162477552/2f9ba913-2b09-4f79-bf64-44311c87f06a)

**Comment by ogoffart** [comment]
Maybe there is some warning in the output of the qttypes build script.
You can maybe scan the file target/debug/build/qttypes-<hash>/output to see if you see a warning, or use cargo `-vv`  to show output from build scripts.
